queries.sql

```diff
@@ -96,4 +96,58 @@ JOIN owners ON owners.id = animals.owner_id WHERE owners.full_name = 'Dean Winch
 -- Who owns the most animals ?
 
 SELECT O.full_name AS OWNER, COUNT(a.name) AS TOTAL_ANIMALS FROM owners AS O LEFT JOIN animals AS A
-ON A.owner_id = O.id GROUP BY O.full_name ORDER BY COUNT(A.name) DESC LIMIT 1; 
\ No newline at end of file
+ON A.owner_id = O.id GROUP BY O.full_name ORDER BY COUNT(A.name) DESC LIMIT 1;
+
+-- Who was the last animal seen by William Tatcher ?
+
+SELECT vets.name AS vets_name, animals.name AS animal_name, visits.date_of_visit AS date_of_visit FROM vets
+INNER JOIN visits ON vets.id = visits.vet_id INNER JOIN animals ON animals.id = visits.animal_id WHERE vets.name = 'William Tatcher'
+ORDER BY visits.date_of_visit DESC LIMIT 1;
```

Comment:
- You have done well so far, but the answer in the screenshot to for this query is wrong, please fix this. The last animal to visit William Tatcher should be `Blossom`. Please fix this 🙏. I suggest you carefully put in your data in the `data.sql`.